src/patternfly/components/InlineEdit/inline-edit.scss

```diff
@@ -0,0 +1,154 @@
+.pf-c-inline-edit {
+  // Group
+  --pf-c-inline-edit__group--item--MarginRight: var(--pf-global--spacer--sm);
+
+  // Valid action
+  --pf-c-inline-edit__action--c-button--m-valid--m-plain--Color: var(--pf-global--link--Color);
+  --pf-c-inline-edit__action--c-button--m-valid--m-plain--hover--Color: var(--pf-global--link--Color--hover);
+
+  // Icon button
+  --pf-c-inline-edit__action--m-icon-button--MarginRight: 0;
+
+  // Toggle
+  --pf-c-inline-edit--m-active__toggle--Color: var(--pf-global--Color--100);
+
+  // Footer group
+  --pf-c-inline-edit__group--m-footer--MarginTop: var(--pf-global--spacer--xl);
+
+  // Action group spacing
+  --pf-c-table__action--pf-c-inline-edit__group--m-action-group--MarginTop: var(--pf-global--spacer--md);
+
+  // Label
+  --pf-c-inline-edit__label--m-bold--FontWeight: var(--pf-global--FontWeight--semi-bold);
+}
+
+// Group
+.pf-c-inline-edit__group {
+  display: flex;
+  align-items: baseline;
+
+  > * {
+    margin-right: var(--pf-c-inline-edit__group--item--MarginRight);
+  }
+
+  // Footer
+  &.pf-m-footer {
+    margin-top: var(--pf-c-inline-edit__group--m-footer--MarginTop);
+  }
+
+  // Column
+  &.pf-m-column {
+    --pf-c-inline-edit__group--item--MarginRight: 0;
+
+    flex-direction: column;
+  }
+
+  // Enable groups
+  &.pf-m-enable {
+    display: inline-flex;
+    visibility: visible;
+
+    .pf-c-inline-edit__action {
+      display: inline-block;
+      visibility: visible;
+    }
+  }
+
+  > :last-child {
+    --pf-c-inline-edit__group--item--MarginRight: 0;
+  }
+}
+
+// Table action cells and action groups inside input groups
+.pf-c-inline-edit .pf-c-table__action,
```

Comment:
where does this selector contain `.pf-c-inline-edit__group.pf-m-action-group`? I'm not sure where this is used.

Reply:
Replaced by `pf-m-icon-group`